### DISCUSSION TITLE
Seeds not executing

### DISCUSSION THREAD

**Discussion by gutyerrez** [adonisjs/core#1268] [Help] [answered]
I tried to run my seeds executing `node ace db:seed` but, after finish i got to my table of categories and nothing have been inserted

## Package version
5.0.0-preview-rc

## Node.js and npm version
Node.js: 12.16.1, NPM: 6.14.6

## Sample Code (to reproduce the issue)
```import BaseSeeder from '@ioc:Adonis/Lucid/Seeder'
import Database from '@ioc:Adonis/Lucid/Database'

export default class CreateCategorySeeder extends BaseSeeder {
  public async run () {
    Database.table('categories').insert([
      { name: 'Announcements' },
      { name: 'Support' },
      { name: 'Community' },
    ])
  }
}
```

**Comment by RomainLanz** [comment, marked as answer]
Hey @Gutyerrez! 👋 

The call `Database.table('categories').insert()` returns a promise. You need to await it.

```ts
await Database.table('categories').insert([
  { name: 'Announcements' },
  { name: 'Support' },
  { name: 'Community' },
])
```

**Comment by gutyerrez** [comment]
Thanks!
